arquillian-liberty-managed-jakarta-junit/pom.xml

```diff
@@ -3,11 +3,11 @@
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
 		<groupId>io.openliberty.arquillian</groupId>
-		<artifactId>arquillian-liberty-dependencies</artifactId>
-		<version>1.1.7-SNAPSHOT</version>
+		<artifactId>arquillian-liberty-jakarta-dependencies</artifactId>
```

Comment:
I see you moved the pom.xml into a differently named folder. This is not consistent with what was done in liberty-arquillian repo.

Reply:
I've reverted the renaming to stay consistent with the liberty-arquillian repo. Only the artifact ids have changed to use the `-jakarta-` suffix. The folder names have stayed the same.